central/development/service/service_impl.go

```diff
@@ -60,7 +63,7 @@ type serviceImpl struct {
 	sensorConnectionManager connection.Manager
 	imageDatastore          imageDatastore.DataStore
 	riskManager             riskManager.Manager
-	client                  http.Client
+	client                  *retryablehttp.Client
```

Comment:
could keep it as a default http client by using `client.StandardClient()` (see [here](https://github.com/hashicorp/go-retryablehttp?tab=readme-ov-file#getting-a-stdlib-httpclient-with-retries)), this way we don't leak library specific client here.